examples/with-solana/README.md

```diff
@@ -166,7 +166,138 @@ Token balance for warchest: 0.0001
 
 Enjoy!
 
-### 6/ Stackblitz Example
+### 6/ Running the "Create SPL token transfer policy" example
```

Comment:
tiny nit: the reason I would include the `with` preposition here is that this example contains more than just creating a policy — we're creating a policy and then using it in practice :)